README.md

```diff
@@ -57,10 +38,134 @@ The following commands are for Ubuntu Server. However, equivalent commands shoul
    * Debug Build: `cargo build`
    * Release Build: `cargo build --release`
    * Run Tests: `cargo test`
-   * Run Server: `cargo run --bin modelardbd path_to_data_folder`
+   * Run Server: `cargo run --bin modelardbd path_to_local_data_folder`
    * Run Client: `cargo run --bin modelardb [server_address] [query_file]`
 5. Move `modelardbd` and `modelardb` from the `target` directory to any directory.
 
+## Usage
+`modelardbd` supports two execution modes *edge* and *cloud*. For storage, `modelardbd` uses local storage and an Amazon S3-compatible object store (optional in edge mode). The execution mode dictates where queries are executed, when `modelardbd` is deployed in edge mode it executes queries against local storage and when it is deployed in cloud mode it executes queries against the object store. For both deployment modes, `modelardbd` automatically compresses the ingested time series using multiple different types of models and continuously transfers this compressed representation from local storage to the object store. Be aware that sharing metadata between multiple instances of `modelardbd` is currently under development, thus only the instance of `modelardbd` that ingested a time series can currently query it.
```

Comment:
Maybe move the `Structure` section up so `modelardb` and `modelardbd` are explained before this. This can be ignored if the naming convention is standard enough to justify it not being explained.

Reply:
That `modelardb` is the client and `modelardbd` is the server is already explained under Installation/All. Also for users [familiar with Unix-like operating systems the naming convention should be familiar](https://en.wikipedia.org/wiki/Daemon_(computing)) as they are used to interacting with programs like `ssh` and `sshd`.
I put Usage before Structure as Structure seemed more relevant for developers than users, so I tried to put it as close to Development and Contributions.